java/kafka/consumer/src/main/java/KafkaConsumerExample.java

```diff
@@ -27,43 +27,45 @@ public static void main(String[] args) {
         int receivedMsgs = 0;
 
         TracingSystem tracingSystem = config.getTracingSystem();
-        if (tracingSystem != TracingSystem.NONE) {
-            if (tracingSystem == TracingSystem.JAEGER) {
-                TracingInitializer.jaegerInitialize();
+        if (tracingSystem != null) {
```

Comment:
When would this be null? Also, why if this is null do we skip the messages? I would expect this to be either NONE or JAEGER or OPENTELEMETRY.